solidity/ERC20ForSPL/openzeppelin-fork/contracts/proxy/ERC1967/ERC1967Proxy.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v5.0.0) (proxy/ERC1967/ERC1967Proxy.sol)
+
+pragma solidity ^0.8.20;
+
+import {Proxy} from "../Proxy.sol";
+import {ERC1967Utils} from "./ERC1967Utils.sol";
+
+/**
+ * @dev This contract implements an upgradeable proxy. It is upgradeable because calls are delegated to an
+ * implementation address that can be changed. This address is stored in storage in the location specified by
+ * https://eips.ethereum.org/EIPS/eip-1967[EIP1967], so that it doesn't conflict with the storage layout of the
+ * implementation behind the proxy.
+ */
+contract ERC1967Proxy is Proxy {
```

Comment:
It seems that this `ERC1967Proxy` isn't used anywhere.

Reply:
Correct, I forgot to remove it from the forked lib, I will remove it.

Reply:
@s-medvedev sorry I was wrong about this! I need this contract for the [deployment scripts](https://github.com/neonlabsorg/neon-tutorials/blob/erc20forspl/make-implementations-editable-by-the-owner/ERC20ForSPL/scripts/deployERC20ForSPLFactory.js#L24) and in the [tests](https://github.com/neonlabsorg/neon-tutorials/blob/erc20forspl/make-implementations-editable-by-the-owner/ERC20ForSPL/test/ERC20ForSPL.js#L53). Hardhat takes the contract and generates the needed artifact. Please check the links.